buildenv/jenkins/variables/defaults.yml

```diff
@@ -410,25 +463,30 @@ osx_x86-64:
   boot_jdk:
     8: '/Users/jenkins/bootjdks/adoptojdk-java-8'
     11: '/Users/jenkins/bootjdks/adoptojdk-java-11'
+    12: '/Users/jenkins/bootjdks/adoptojdk-java-11'
     next: '/Users/jenkins/bootjdks/adoptojdk-java-11'
   release:
     8: 'macosx-x86_64-normal-server-release'
     11: 'macosx-x86_64-normal-server-release'
+    12: 'macosx-x86_64-normal-server-release'
     next: 'macosx-x86_64-server-release'
   extra_configure_options:
     8: '--with-noncompressedrefs --with-xcode-path=/Users/jenkins/Xcode4/Xcode.app --with-openj9-cc=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang --with-openj9-cxx=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ --with-openj9-developer-dir=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer'
     11: '--with-noncompressedrefs --with-openssl=fetched --enable-openssl-bundling'
+    12: '--with-noncompressedrefs --with-openssl=fetched --enable-openssl-bundling'
```

Comment:
I must have merged something incorrectly. 12 and next do not support openssl yet and should not have openssl parameters. This applies to all platforms.